loadflow/loadflow-api/src/main/java/com/powsybl/loadflow/json/LoadFlowParametersDeserializer.java

```diff
@@ -62,6 +62,12 @@ public LoadFlowParameters deserialize(JsonParser parser, DeserializationContext
                     parameters.setTransformerVoltageControlOn(parser.readValueAs(Boolean.class));
                     break;
 
+                case "useReactiveLimits":
+                    JsonUtil.assertGreaterOrEqualThanReferenceVersion(CONTEXT_NAME, "Tag: useReactiveLimits", version, "1.8");
+                    parser.nextToken();
+                    parameters.setUseReactiveLimits(parser.readValueAs(Boolean.class));
+                    break;
+
                 case "noGeneratorReactiveLimits":
```

Comment:
Can you assert lower or equal than 1.7 when noGeneratorReactiveLimits is read?

Reply:
Done, there was a bug because of this missing check :-)